src/templates/EditorialNewsletter/schema.js

```diff
@@ -250,6 +251,32 @@ const createNewsletterSchema = ({
                   formatButtonText: 'Zwischentitel'
                 }
               },
+              {
+                matchMdast: matchZone('BUTTON'),
+                component: Button,
+                props: (node, index, parent, { ancestors }) => {
+                  const link = (
+                    node.children[0] &&
+                    node.children[0].children[0]
+                  ) || {}
+
+                  return {
+                    title: link.title,
+                    href: link.url,
+                    primary: node.data.primary
+                  }
+                },
+                rules: globalInlines.concat({
+                  matchMdast: matchParagraph,
+                  component: ({ children }) => children,
+                  rules: [{
+                    matchMdast: matchType('link'),
+                    component: ({ children }) => children,
+                    rules: globalInlines
+                  }]
+                }),
+                editorModule: 'button'
+              },
```

Comment:
This piece of configuration is so similar to the one in the article schema, I am still in doubt regarding whether it  would make sense to have a setButtonSchema function (or similar) somewhere that takes the component as argument and use that instead.